packages/browser/src/client.ts

```diff
@@ -26,7 +26,26 @@ import { createUserFeedbackEnvelope } from './userfeedback';
  */
 export type BrowserOptions = Options<BrowserTransportOptions> &
   BrowserClientReplayOptions &
-  BrowserClientProfilingOptions;
+  BrowserClientProfilingOptions & {
+    /**
+     * Important: Only set this option if you know what you are doing!
+     *
+     * By default, the SDK will check if it is initialized in a browser extension
```

Comment:
L: "By default, the SDK will check if `Sentry.init` is called in a browser extension."